torchao/experimental/kernels/mps/src/lowbit.h

```diff
@@ -125,6 +125,7 @@ void linear_lowbit_quant_weights_mps(
     int32_t N,
     const std::string_view type_str) {
   assert(K % 8 == 0);
```

Comment:
I dont know if this is a sufficient condition for optimized operators. Because when k is not multiple of what thread group handles per iteration, I dont know what we do. Or we probably do something wrong? Or maybe not because each thread in the threadgroup handles at most 8 k values per iteration.

Reply:
yeah makes sense